_includes/helper_action_functions.md

```diff
@@ -0,0 +1,25 @@
+
+|Name          |Type      |Description
+|-----------    |-----------|--------------
+|add_tags      |strings[]     |An array of tag names corresponding to previously created tags to add to this person when it is created.
+|add_tags_uri  |strings[]     |An array of tag URIs corresponding to previously created tags to add to this person when it is created.
+|add_lists     |strings[]     |An array of list names corresponding to previously created lists to add to this person when it is created.
+|add_lists_uri     |strings[]     |An array of list URIs corresponding to previously created lists to add to this person when it is created.
+|add_questions_responses_uri      |[AnswerActionFunction](#answer-action-function-fields)     |An array of Person Signup Helper Answer objects, indicating answers which should be applied to the Person at the moment of signup
+|triggers		|[Triggers](#triggers)	|An object hash representing responses a user would like to trigger from the server as part of the POST, such as sending an autoresponse email back to the person who took action with this helper.
+
+### Answer Action Function Fields
```

Comment:
Shouldn't they be Question Answers, since we are giving answers?  Responses are what we describe in a definition of a question meaning the possible answers?